packages/drivers/odsp-driver/src/contractsPublic.ts

```diff
@@ -56,8 +56,27 @@ export interface IClpCompliantAppHeader {
 	[ClpCompliantAppHeader.isClpCompliantApp]: boolean;
 }
 
+/**
+ * @internal
+ */
+export enum IfMatchMigrationHeader {
```

Comment:
No need to use the name IfMatch. This is just eTag. 
IfMatch is just the header we will set when making the create request and this is internal detail. We can just call this FileMetaDataHeader with eTag inside it. So, in future we can add more values to it.